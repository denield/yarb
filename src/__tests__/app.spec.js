import React from 'react'
const renderer = require('react-test-renderer')
import App from '../app'

describe('<App/>', () => {
  it('renders hello world', () => {
    const tree = renderer
      .create(<App/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})