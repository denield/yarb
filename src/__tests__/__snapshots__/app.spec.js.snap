// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> renders hello world 1`] = `
<div>
  Hello World
</div>
`;
